src/openrct2/game.c

```diff
@@ -1083,10 +1083,12 @@ bool game_load_save(const utf8 *path)
     bool result = false;
     if (extension_type == FILE_EXTENSION_SV6) {
         result = game_load_sv6_path(path);
+        if (result)
```

Comment:
We left out this call deliberately here. When the user has loaded from an SV6, it's okay to overwrite that during the first quick save, while you can't do that with an SV4 file.